### ISSUE TITLE
Improve message for users if HTML report cannot be created from CTL

### ISSUE THREAD

**Issue by dstenger** [opengeospatial/teamengine#230]
* The alert should not appear when the HTML report is present.
* Suggestion: Can the alert be replaced by a mechanism which just changes the display of the HTML report to the text currently written in the alert?
E.g. following is displayed on the result page (no alert):
```
This test suite does not support the new HTML report and the XML report must be used instead.

See the detailed old test report.
```

**Comment by keshav-nangare** [comment]
Fixed with this commit: 1be43083afe580c689551f1465822bc25987defe

**Comment by dstenger** [comment]
= Test

Test with ets-wms13 and [1]. An alert is shown as described above and the old report can be accessed.
Test with ets-wfs20 and [2]. New HTML report is shown.

New reporting format if HTML report is not supported has the expected design.

[1] http://cite.deegree.org/deegree-webservices-3.4-RC3/services/wms130?service=WMS&request=GetCapabilities
[2] http://cite.deegree.org/deegree-webservices-3.4-RC3/services/wfs200?service=WFS&request=GetCapabilities
